Platform/Microsoft/Application/AuthVarTest/AuthVarTest.c

```diff
@@ -137,17 +137,18 @@ VerifyVariableEnumerable (
       VERIFY_IS_NOT_NULL (mEnumVariableName);
       VariableNameBufferSize = VariableNameSize;

```

Comment:
You should include test cases for the following:
Different length variable names, what is the max length?
Different cased variable names (are they case sensitive?)
Variable names with / without accents (  a vs à )
Variable names with special characters.
etc.